projects/ADuCM3029_demo_cn0548/builds.json

```diff
@@ -0,0 +1,3 @@
+{
+    "cn0548": ""
```

Comment:
nitpick: I would use iio or demo as name and then the final .hex name will be: ADuCM3029_demo_cn0548_iio.
not ADuCM3029_demo_cn0548_cn0548

Reply:
cool, will do.